packages/amplication-data-service-generator/src/util/ast.ts

```diff
@@ -769,3 +779,78 @@ export function createGenericArray(
     builders.tsTypeParameterInstantiation([itemType])
   );
 }
+
+export function removeDecoratorByName(
```

Comment:
why do we still need this function?

Reply:
Maybe I got you wrong. Your comment was something like "create a specific version of this function to interceptors", so I thought you mean to leave as a function for decorator without namespace and  create a new specific one for interceptors

Reply:
removeDecoratorByName is used to remove UseRoles decorator